Sources/Typology/AST/SwiftSyntax.swift

```diff
@@ -19,11 +19,12 @@ extension Array where Element == Statement {
       try statement.children.flatMap { syntax -> [Statement] in
         switch syntax {
         case let sequence as SequenceExprSyntax:
-          return try sequence.elements.map(Expr.init)
+          return try sequence.elements.map { ExprNode(position: $0.position, expr: try Expr($0)) }
 
         case let function as FunctionDeclSyntax:
           let returns = function.signature.output?.returnType
           let body = function.body?.statements
+          let position = function.body?.position
```

Comment:
this calculates position of a function body, but not a position of the whole function. Please replace this with `function.position`